src/main/java/ca/uwaterloo/drinkmasterapi/sql/creation/drink.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE drink (
+                       id BIGINT AUTO_INCREMENT PRIMARY KEY,
+                       name VARCHAR(255) NOT NULL,
+                       categoryId BIGINT NOT NULL,
+                       imageURL VARCHAR(255),
+                       price DOUBLE NOT NULL,
+                       priceCurrency VARCHAR(255) NOT NULL,
+                       description VARCHAR(255),
+                       isActive BOOLEAN DEFAULT true,
+                       isCustomized BOOLEAN DEFAULT false,
+                       userId BIGINT,
```

Comment:
if it is customize, user id is user id; if it is not customized, user id means admin id, just let u know, no changes need here